cccm-backend/cccm-api/src/main/java/ca/bc/gov/open/jag/api/mapper/UserMapper.java

```diff
@@ -50,10 +50,6 @@ public interface UserMapper {
     @Mapping(target = "dueSeven", source = "notRequired")
     SupervisorDashboardDetails toSupervisorDashboardDetails(ca.bc.gov.open.jag.api.model.data.SupervisorDashboardDetails supervisorDashboardDetails);
 
-    @Mapping(target = "idirId", source = "idirId")
```

Comment:
Mappstruct will map fields that are the same. I usually like to be explicit to show all things being mapped. However in this case they are all equal so I figured to just leave them out. But I can add them in if it seems strange to leave them out in one case.